.vscode/settings.json

```diff
@@ -1,7 +1,5 @@
 {
-	"editor.tabSize": 4,
-	"editor.detectIndentation": false,
-	"editor.insertSpaces": false,
```

Comment:
if i understand correctly what `insertSpaces` does, I'd prefer for it to stay there

Reply:
oh, wait. it's VSCode config not tslit file. nevermind, I don't have preferences here (as long as we actually have the spaces in the codebase :p)